src/features/lightspeed/utils/webUtils.ts

```diff
@@ -71,21 +71,21 @@ export function getBaseUri(settingsManager: SettingsManager) {
 }
 
 export function getLoggedInSessionDetails(
-  sessionData: LightspeedAuthSession
+  sessionData?: LightspeedAuthSession
```

Comment:
It would be more elegant create a dummy  `LightspeedAuthSession` object in `getSessions()` when we are this context. This way we don't need to relax the type-checking to support a corner case.

Reply:
I have tried to implement the logic as you suggested, but could not make it work. As the issue was found even with the actual Lightspeed server and irrevant to the mock server implementation. I think we can handle it in a separate issue. I reverted the changes on the file and will open a new issue.

Reply:
#1063 was opened for the issue.